### DISCUSSION TITLE
SQLite to optimize static builds

### DISCUSSION THREAD

**Discussion by gaboesquivel** [vercel/next.js#67570] [Help]
### Summary

I currently have a growing number of static files used at build time, I think moving the data to sqlite is a good option to reduce build time. Do you have any other suggestions?  perhaps I'm missing a new tool

**Comment by BorhanSaflo** [comment]
I recommend using a database instead of static files to store your data. You can use an ORM like [Prisma](https://www.prisma.io/) to simplify database interactions. Here's a [guide](https://dev.to/azadshukor/setting-up-nextjs-13-with-prisma-and-sqlite-1lld) on how to set it up with SQLite and Next.js.

**Reply by gaboesquivel** [reply]
Thanks, I'm doing automated internationalization and saving all dictionaries to files, we run a script to read from cms the English version
